### DISCUSSION TITLE
Github Actions Ideas

### DISCUSSION THREAD

**Discussion by j0n3lson** [mellychat/webapp#77] [Ideas]
* Document freshness (ping if a doc hasn't been touched),
* Mark a PR is blocked by another

**Comment by Mannendri** [comment]
- Auto-add the no-pr-activity label to inactive pull requests

**Reply by j0n3lson** [reply]
Doe it not do that already? I feel like it should do that :)

**Reply by Mannendri** [reply]
Oh, it does! I thought that it only did the ping. Never mind then.
